### PR TITLE
Wrappingup labs

### PR DESCRIPTION
**Pull request by Fellylove**
# Description
This pull request is focused on wrapping up labs. It readme files and documentation. I have also included some notebooks and scripts that will make onboarding easier for the next cohort. 

https://www.loom.com/share/59d085c6bb0145d8841124ca8760e0e9

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Change Status

- [ ] Complete, tested, ready to review and merge
- [ ] Complete, but not tested (may need new tests)
- [ ] Incomplete/work-in-progress, PR is for discussion/feedback

## Has This Been Tested

- [ ] Yes
- [ ] No
- [x] Not necessary

## Checklist

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [x] There are no merge conflicts

### CODE REVIEW

**Review by ike-okonkwo — APPROVED** [review]
This looks great. Thanks for updating readmes, documentation, and results obtained

**Review by techthumb1 — APPROVED** [review]
Great work Sylvia. Well written, code is clean, and definitely a roadmap for the following cohort, great job!

File: custom_tesseract_training/README.md [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+      command again and model will start from where an error was made. When the model gets to this point
+      `At iteration 98/100/100, Mean rms=3.128%, delta=14.449%, char train=56.768%, word train=74.568%, skip ratio=0%, New best char error = 56.768 
+      wrote best model:data/storysquad/checkpoints/storysquad56.768_98.checkpoint wrote checkpoint.
+      Loaded 1/1 lines (1-1) of document data/storysquad-ground-truth/5220-1-014.lstmf
+      Loaded 1/1 lines (1-1) of document data/storysquad-ground-truth/5220-1-014.lstmf`
+      It has gone through all the files. There shouldn't any file related errors at this point.
+    - once the model has finished training, the new model will be stored in `tesstrain\data`. The default 
+      name right now is `storysquad.traineddata`. New models will overwrite the old ones if they are not not
+      moved to another directory or renamed.
+
```
Comment (BritVandy) [inline custom_tesseract_training/README.md:46]: Really well documented and explained.

File: custom_tesseract_training/ocr_inference.py [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+    
+default = pytesseract.image_to_string(image)
+custom = pytesseract.image_to_string(image, lang='storysquad')
+
+print(default)
+print(custom)
+
+## compare accuracy of defualt model and fine -tuned model  
+
+# read text from the ground-truth file
```
Comment (BritVandy) [inline custom_tesseract_training/ocr_inference.py:20]: Where is this pulling the document from? It looks like it’s pulling from a local location.

**Review by BritVandy — COMMENTED** [review]
Overall good documentation.

File: custom_tesseract_training/ocr_inference.py (hunk 2 above)
Comment (Fellylove) [reply custom_tesseract_training/ocr_inference.py:20]: You run this in the EC2 instance. It pulls the document from 'data' dir. I explained it in the Readme.

Comment (Fellylove) [reply custom_tesseract_training/ocr_inference.py:20]: I think I do need to fix line 21 though. Thanks for pointing that out @BritVandy

**Pull request merged**